chatkit-core/src/main/kotlin/com/pusher/chatkit/state/ChatkitState.kt

```diff
@@ -0,0 +1,7 @@
+package com.pusher.chatkit.state
+
+import com.pusher.chatkit.rooms.state.JoinedRoomsState
+
+internal data class ChatkitState(
+    val joinedRoomsState: JoinedRoomsState
```

Comment:
```suggestion
    val joinedRoomsState: JoinedRoomsState?
```
per our breakfast chat :)

Reply:
ah of course! Thanks for following up!